test/functional/wallet_import_rescan.py

```diff
@@ -116,6 +134,14 @@ def check(self, txid=None, amount=None, confirmations=None):
 # Rescans start at the earliest block up to 2 hours before the key timestamp.
 TIMESTAMP_WINDOW = 2 * 60 * 60
 
+AMOUNT_DUST = Decimal('0.00000546')
+
+
+def get_rand_amount():
```

Comment:
This returns `None` if the random value returned is less than `AMOUNT_DUST`.

Why not use `random.uniform(AMOUNT_DUST,1)` rather than define this function?

Reply:
Switched to uniform

Reply:
Switched to uniform